### DISCUSSION TITLE
preparing training data

### DISCUSSION THREAD

**Discussion by rushboozoo** [acids-ircam/RAVE#142] [Q&A]
I got the error message 'audio amplitude out of range, auto clipped.' and I don't understand why this happened. 
I have a five hour training set with musical data. The material is limited to -6dB and has a dynamic range of approx. 13 LUFS.
Are there any experiences on how to prepare the audio files?
I tried to limit the dynamic cause my models where strongly responding to the input volume and I was not so happy with this behavior.

Thanks in advance

**Comment by CaMeLCaseError** [comment]
I'm pretty sure this is only concerning the generation of audio validations. In case you didn't use the audio validations yet: you can listen to the audio validation e.g. in tensorboard. I have seen this message now pretty often and it is always appearing in the moment a new validation was made. So don't worry about your input data... :)
nice regards, Martin

**Comment by devstermarts** [comment]
@rushboozoo do you have any experience to share on training with same data in different dynamics (e.g. fully processed/ mastered vs. unprocessed/ mixed material)? I'd be interested in differences in the models' characters and viability. 

I'm currently experimenting with training data sets of mastered electronic music which are quite loud and have hardly any dynamics any more. The models seem to reproduce that but (depending on the input of course) create quite muddy sounding, not very dynamic results. 

Maybe @moiseshorta has some insights on this as well?
